faq.html

```diff
@@ -218,14 +218,22 @@ <h3>What about serving specs using a third-party tool? <a href="#others">&#x1F51
         documents with the appropriate MIME type, and from more than one branch (because of
         <a href="#drafts">these limitations of GitHub Pages</a>).
         That service (RawGit) is now defunct.</p>
-      <p>Nowadays, if you absolutely need such a service, we would probably suggest
-        <a href="https://www.staticaly.com/">Staticaly</a> for the same purpose&nbsp;&mdash;&nbsp;with
-        the caveat that <em>it is just a recommendation that comes with no support nor guarantee</em>,
-        since we do <em>not</em> control the service. (The reason to opt for this service is that
-        others, like <a href="https://htmlpreview.github.io/">HTMLPreview</a> or
-        <a href="https://raw.githack.com/">raw.githack.com</a>, either rely on client-side JavaScript
-        to generate the page dynamically, are limited to GH repositories only, or alter the pages or
-        the links within them in some way.)</p>
+      <p>Nowadays, if you absolutely need such a service, we would probably suggest the following two options
+        &nbsp;&mdash;&nbsp;with the caveat that <em>it is just a recommendation that comes with no support nor
+          guarantee</em>, since we do <em>not</em> control these services:</p>
+        <ul>
+          <li><a href="https://www.staticaly.com/">Staticaly</a>. By default, staticaly's cache is set for
+            <em>1 year</em>, except for the files under the <code>master</code> branch. If you need the latest
+            changes, you may append the following query string <code>?env=dev</code> to the URL.
+          <li><a href="https://raw.githack.com/">raw.githack.com</a>. The UI of this service gives you URLs for
+            production and development you can use depending on your needs. <strong>Note</strong>: this service
+            is ran by a single individual and it does warn about the traffic generated by development URLs in
```

Comment:
As we discovered that **Statically** does not seem to have mush more contributors than **githack.com** I would suggest removing teh following bit: `is ran by a single individual and it`

Reply:
adressed!